readme.md

```diff
@@ -1,5 +1,18 @@
 # Web Accessibility hands-on Quintor
 
+## Instructies om het project starten:
+
+- Kloon het project <br/> `git clone https://github.com/Quintor/workshop-accessibility.git`
+- Ga naar de projectdirectory <br/> `cd workshop-accessibility`
+- Installeer de benodigde afhankelijkheden <br/> `npm install`
+- Start het project <br/> `npm start`
+
+### Als je komt tegen een probleem met de installatie van de afhankelijkheden, probeer dan de volgende stappen:
+
+- Verwijder de `node_modules` directory <br/> `rm -rf node_modules`
+- Verwijder het `package-lock.json` bestand <br/> `rm package-lock.json`
```

Comment:
Ik zou de rm -rf weglaten, die maakt het mogelijk verwarrend en is alleen toepasbaar op linux vgm